### PR TITLE
Fb/osmocom working

### PR DESCRIPTION
**Pull request by kheimerl**
contains fixes to #58  and an issue with the openvpn package. With this in place (and the other workarounds) I got an osmocom vm up and running.

### CODE REVIEW

File: client/fabfile.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def package(package_requirements='yes', flavor=''):
         package_python_openbts(package_requirements)
     elif (env.gsmeng == "osmocom"):
         package_python_osmocom(package_requirements)
-        package_freeswitch_mod_smpp()
+        #package_freeswitch_mod_smpp()
```
Comment (9muir) [inline client/fabfile.py]: @kkroo, I think this just needs `s/freeswitch_mod_smpp/freeswitch/`, correct?

Comment (kkroo) [inline client/fabfile.py]: It should be fine to remove from the package command. Most people will not need to rebuild freeswitch in their development flow

Comment (9muir) [inline client/fabfile.py]: Makes sense. Let's just remove the call altogether then, not comment it out.

File: client/conf/registration/openvpn.conf [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 directory=/etc/openvpn
 
-command=openvpn endaga-vpn-client.conf.noauto
+command=/usr/sbin/openvpn endaga-vpn-client.conf.noauto
```
Comment (9muir) [inline client/conf/registration/openvpn.conf]: 👍

**Review by 9muir — CHANGES_REQUESTED** [review]
The python_packaging changes are logically subsumed by the internal packaging diff I just created, but the OpenVPN change is helpful.

**Comment by kheimerl** [comment]
So should I convert this to include just the removal of the smpp build and the openvpn change?

**Comment by 9muir** [comment]
confirm that the change I pushed works for you too (I did test, but just to be sure). I'm happy with either removing the smpp build or leaving it as it now is, whichever you and @kkroo prefer, just don't have something commented out. The OpenVPN change is the most useful.

**Comment by kheimerl** [comment]
i can't test the diff, I work off the open source repo :( I'l let omar call
that ball.

**Comment by kkroo** [comment]
I like what @9muir did in making freeswitch build only if you have the source checkout out. Let's use the internal diff. Closing this out

**Comment by 9muir** [comment]
@kheimerl the change is pushed to the open source repo as 443fa69

**Pull request closed without merging**